storage/storage-client/src/lib.rs

```diff
@@ -431,6 +437,25 @@ pub trait StorageRead: Send + Sync {
     ) -> Result<BoxStream<'_, Result<BackupTransactionInfoResponse, Error>>>;
 }
 
+impl ConfigStorage for Box<&dyn StorageRead> {
+    fn fetch_config(&self, access_path: AccessPath) -> Option<Vec<u8>> {
+        let account_state_blob = async {
+            self.get_latest_account_state(access_path.address)
+                .await
+                .ok()?
+        };
+        let result = block_on(account_state_blob);
```

Comment:
just fyi, I spend a day debug why this client would stuck with block_on and didn't figure out. fortunately we're deprecating this, you probably can implement this on LibraDBTrait instead